src/photod/column_map/glossary.yaml

```diff
@@ -0,0 +1,134 @@
+# Glossary of terms for this project
```

Comment:
This has more fields than are used in the code. Is this just waiting to be cleaned up?